lib/arrow_web/controllers/shape_html.ex

```diff
@@ -1,5 +1,9 @@
 defmodule ArrowWeb.ShapeView do
   use ArrowWeb, :html
+  alias Arrow.Shuttles
+  alias Arrow.Shuttles.Route
+  alias Arrow.Shuttles.RouteStop
+  alias Arrow.Shuttles.Shape
   alias Arrow.Shuttles.ShapesUpload
```

Comment:
suggestion: These could be grouped
```suggestion
  alias Arrow.Shuttles.{Route, RouteStop, Shape, ShapesUpload}
```